File: backend/Models/Route.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace backend.Models;

[Table("routes")]
public class Route
{
    [Column("route_id")]
    [Required]
    public string RouteId { get; set; } = null!;
    
    [Column("agency_id")]
    [Required]
    public string AgencyId { get; set; } = null!;
    
    [Column("route_short_name")]
    [Required]
    public string RouteShortName { get; set; } = null!;
    
    [Column("route_long_name")]
    [Required]
    public string RouteLongName { get; set; } = null!;
    
    [Column("route_desc")]
    [Required]
    public string RouteDesc { get; set; } = null!;
    
    [Column("route_type")]
    [Required]
    public int RouteType { get; set; }
    
    [Column("route_color")]
    [Required]
    public string RouteColor { get; set; } = null!;
    
    [Column("route_text_color")]
    [Required]
    public string RouteTextColor { get; set; } = null!;

    [Column("route_url")]
    [Required]
    public string RouteUrl { get; set; } = null!;
}

File: backend/TransportRealtime/models/route.go
package models

type Route struct {
	RouteId          string `json:"route_id"`
	AgencyId         string `json:"agency_id"`
	RouteShortName   string `json:"route_short_name"`
	RouteLongName    string `json:"route_long_name"`
	RouteDescription string `json:"route_desc"`
	RouteType        int    `json:"route_type"`
	RouteColor       string `json:"route_color"`
	RouteTextColor   string `json:"route_text_color"`
	RouteUrl         string `json:"route_url"`
}
